client/index.html

```diff
@@ -4,6 +4,8 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <title>CELL&times;GENE | Explorer</title>
+    <meta property="og:title" content="CELLxGENE | Explorer" />
+    <meta property="og:image" content="static/images/cxg-explorer-og.jpg" />
```

Comment:
👍🏽 🙏🏽

Reply:
Thanks so much for the review, @kaloster !! 🙏